pages/vi/profiles/aj4ayushjain.md

```diff
@@ -0,0 +1,21 @@
+# About Me
+
+Name | Location/Timezone | OS
+--- | --- | ---
+*Ayush Jain* | Bangalore(India)-GMT+530| **Ubuntu(18.04)**
+
+
+![alt-text](https://img.etimg.com/thumb/msid-72885396,width-643,imgsize-1240914,resizemode-4/randstad-insights-report-also-reveals-that-professionals-from-the-it-industry-command-the-highest-salaries-.jpg "Namma Bengaluru")
+
+
+Hi,<br>
```

Comment:
do not think `<br>` is necessary
... think adding an empty line is doing it too